deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,46 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+body {
+    display: flex;
+    flex-direction: row;
+    height: 100vh;
+}
+
+.outer-col-1 {
```

Comment:
Voor deze kolom mist een flex-grow en flex-shrink van 0. Met een flex-basis in plaats van width had je eerste 2 regels weg kunnen laten